src/main/kotlin/org/sert2521/infiniterecharge2020/autonomous/Paths.kt

```diff
@@ -0,0 +1,63 @@
+package org.sert2521.infiniterecharge2020.autonomous
+
+import edu.wpi.first.wpilibj.controller.RamseteController
+import edu.wpi.first.wpilibj.geometry.Pose2d
+import edu.wpi.first.wpilibj.geometry.Rotation2d
+import org.sert2521.infiniterecharge2020.drivetrain.Drivetrain
+import org.sert2521.infiniterecharge2020.drivetrain.kinematics
+import org.sert2521.infiniterecharge2020.drivetrain.runPath
+import org.sert2521.sertain.subsystems.doTask
+import org.sert2521.sertain.subsystems.use
+import org.sert2521.sertain.units.mps
+
+suspend fun rightInitPowerPort(pushBack: Boolean, endLocation: PathGenerator.endLocation) = doTask {
```

Comment:
This function and the `centerInitPowerPort` function are virtually identical, save for one line. We should combine these into one function and pass in our initial location as a parameter. For example:
```
initPowerPort(true, PathGenerator.startLocation.RIGHT, PathGenerator.endLocation.TRENCH)
```

Reply:
Example of doing that last year:

https://github.com/SouthEugeneRoboticsTeam/DeepSpace-2019/blob/8ebc27674f2ae5d840bfd4a6578bc555a2896f61/src/main/kotlin/org/sert2521/deepspace/autonomous/Autos.kt#L85-L101